drivers/lcd/ssd1680.c

```diff
@@ -126,11 +126,11 @@ struct ssd1680_dev_s
 
 /* Libc extension */
```

Comment:
is below interface really a `Libc extension`? Or it is a copy-paste leftover?

Reply:
OK. I forgot about description. I will fix it.

Reply:
I wrote extended memcpy functions, where we can address bits (not bytes). Libc doesnt have fuction tohat let us to address bits.
I cal remove this comment. I wrote just two functions. I was wandering if add it to libs or not.